#237 customer input

joshxg opened the pull request:
Fixes #227 

![image](https://user-images.githubusercontent.com/35858975/137414732-02be5bfa-44aa-4804-9fcb-1650c619cb69.png)


Not sure if the styles are quite right here but thought to stop here. Will do something very similar for the ref input, so figured heres a good place to stop and get feedback before moving forward.

mark-prins commented on packages/common/src/ui/components/inputs/TextInput/InputWithLabelRow.tsx:
```diff
@@ -0,0 +1,18 @@
+}) => {
+  return (
+    <Grid display="flex" alignItems="center" gap={1}>
+      <FormLabel sx={{ fontWeight: 'bold' }}>Customer name:</FormLabel>
```
was going to point out that it needs a translation, but then noticed this is a shared component.. so will need the label to be passed in

mark-prins commented on packages/invoices/src/OutboundShipment/DetailView/DetailView.tsx:
```diff
@@ -210,12 +212,34 @@ export const OutboundShipmentDetailViewComponent: FC = () => {
+          </TabList>
+        </Box>
+
+        <Box display="flex" flex={1} justifyContent="flex-end" />
```
Is it time to put a save and cancel button on the bottom of this component 🤔

joshxg replied:
Ha yes I think so!

mark-prins commented on packages/mock-server/src/worker/handlers.ts:
```diff
@@ -108,7 +121,7 @@ export const serverError = graphql.query('error500', (_, response, context) =>
  */
 const batchMutationHandler = rest.post(
-  Environment.API_URL,
+  'http://localhost:4000',
```
are you having issues with the config too?

joshxg commented:
Seems what I've added in the MSW to handle batched mutations means the items query wasn't falling through and hitting the server. So i've added an item handler in the worker for now and will look at doing a better solution for the mock worker.

Also fixed up the nits in the other PR in this one
